app/scripts/modules/cloudfoundry/src/pipeline/stages/destroyService/CloudfoundryDestroyServiceExecutionDetails.tsx

```diff
@@ -1,15 +1,44 @@
 import * as React from 'react';
+import { get } from 'lodash';
 
 import {
+  AccountTag,
   ExecutionDetailsSection,
   IExecutionDetailsSectionProps,
   StageExecutionLogs,
   StageFailureMessage,
 } from '@spinnaker/core';
 
 export function CloudfoundryDestroyServiceExecutionDetails(props: IExecutionDetailsSectionProps) {
```

Comment:
This looks just like CloudfoundryDeployServiceExecutionDetails. Any way to share this?

Reply:
👍